pkgs/development/libraries/apr-util/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
 
   configureFlags = ''
     --with-apr=${apr} --with-expat=${expat}
-    --with-crypto
+    ${if !stdenv.isCygwin then "--with-crypto" else "--without-pgsql --without-sqlite2 --without-sqlite3 --without-freetds --without-berkeley-db --without-crypto"}
```

Comment:
Quoting Daniel Peebles (2015-06-11 15:30:22)
> In pkgs/development/libraries/apr-util/default.nix:
> 
> > @@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
> >
> >    configureFlags = ''
> >      --with-apr=${apr} --with-expat=${expat}
> > -    --with-crypto
> > +    ${if !stdenv.isCygwin then "--with-crypto" else "--without-pgsql --without-sqlite2 --without-sqlite3 --without-freetds --without-berkeley-db --without-crypto"}
> 
> Lots of negation in here! "if not cygwin then use crypto, otherwise (not not
> cygwin), I want no pgsql, no sqlite2/3, etc."
> 

i've ported configureFlags from cygport.

i don't want to cause major rebuilds for cygwin related work. i actually make
sure that nothing changes for linux/darwin when pushing to master.

we're far from perfect but little steps like this will get us closer to
something more generaly usable on cygwin.


--
Rok Garbas - http://www.garbas.si